webclient/app/src/app/commons/gitDownloadButton/GitDataButtonStyles.js

```diff
@@ -0,0 +1,25 @@
+import {makeStyles} from "@mui/styles";
+
+const GitDataButtonStyles = makeStyles(theme => ({
+    tabBox: {
+        borderBottom: 1,
+        borderColor: 'divider'
+    },
+    dialogHeaderBar: {
+        display: "flex",
+        alignContent: "center",
+        justifyContent: "space-between"
+    },
+    statementWrapper: {
+        height: "15rem"
+    },
+    flex: {
+        flex: 1
+    },
+
+    checkbox: {
+        margin: "8px",
```

Comment:
please use theme.spacing(4)
```suggestion
        margin: theme.spacing(4),
```
(or an equivalent factor)

Reply:
changed